ui_framework/src/components/side_nav/_side_nav.scss

```diff
@@ -0,0 +1,43 @@
+// These mobile children are hidden in everything but mobile view.
+.kuiSideNav__mobileToggle {
+  display: none;
+  flex-direction: row;
+  border-bottom: $kuiBorderThin;
+  padding: $kuiSize $kuiSizeL;
+  width: 100%;
+  text-align: left;
+}
+
+.kuiSideNav__mobileIcon {
+  fill: $kuiColorPrimary;
+}
+
+.kuiSideNav__mobileTitle {
+  flex-grow: 1;
+  color: $kuiColorPrimary;
+}
+
+@include screenXSmall {
+  /**
+   * 1. Animation doesn't work against height. Need max-height instead.
+   *    We set a value larger than what is needed to fake "auto".
+   */
+  .kuiSideNav.isOpenMobile .kuiSideNav__content {
+    visibility: visible;
+    opacity: 1;
+    margin: $kuiSizeL;
+    max-height: 5000px; /* 1 */
+  }
+
+  .kuiSideNav__mobileToggle {
+    display: flex;
```

Comment:
And once we do that, we can add `justify-content: space-between` to that element, and remove `flex-grow: 1` from `.kuiSideNav__mobileTitle`.